config/config.local.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Service__name": "pickaxe",
+  "Service__host": "http://pickaxe:8050"
```

Comment:
Would be nice to have a directory somewhere that automatically pulls all the ports for the services being run (diamond, pickaxe, sapphire, etc)

Reply:
yeah thats true, definetly in the plan for improving devtools, most likely going to be an internal KV store for all the configs from where devtools is going to pull from, this will also save us from sending the config.yml file to each other